composeApp/src/androidMain/kotlin/org/ooni/probe/AndroidApplication.kt

```diff
@@ -81,6 +84,17 @@ class AndroidApplication : Application() {
         }
     }
 
+    override fun onCreate() {
+        super.onCreate()
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+            val localeManager = applicationContext
+                .getSystemService(LocaleManager::class.java)
+            localeManager.overrideLocaleConfig = LocaleConfig(
+                LocaleList.forLanguageTags(getString(R.string.supported_languages)),
+            )
+        }
```

Comment:
Why do we need to set the supported languages here as well? Isn't the code on `build.gradle` enough?
```
        resourceConfigurations += config.supportedLanguages
```

Reply:
I checked it out and it doesnt actually enable the languages in the settings.